Modules/Quest/QuestEventHandler.lua

```diff
@@ -70,6 +73,28 @@ function _QuestEventHandler:InitQuestLog()
         return false
     end
 
+    for i = 1, MAX_QUEST_LOG_INDEX do
```

Comment:
This loop should be merged with the the loop below. Both are partly doing the same

Reply:
We will keep these separate to make the code more readable. They are also run only on addon load so performance does not matter.